liquigraph-core/src/test/resources/changelog/changelog-with-2-nodes.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright 2014-2020 the original author or authors.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+         http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<changelog>
+    <changeset id="insert-fbiville" author="fbiville">
+        <query><![CDATA[CREATE (:Human {name:'fbiville'})]]></query>
```

Comment:
optional: you can remove `<![CDATA[` and `]]>` given there is no `>` or `<` in the query contents (that usually happens when matching relationships)

Reply:
Most of the existing changelogs in this folder use CDATA in the same condition (no < and > in the query).

The best imo will be to update all the changelogs to be consistent. 

In the meantime, I'll leave this as-is if it's ok for you.